src/fu-util.c

```diff
@@ -2449,6 +2449,12 @@ main (int argc, char *argv[])
 			    "is no longer supported by the upstream developers!\n");
 	}
 
+	/* turn on daemon logging potentially */
+	if (verbose && !fwupd_client_enable_verbose (priv->client,
```

Comment:
I think splitting into two if's would be clearer here.

Reply:
done